src/expo-plugin/withDangerouslyHandleAndroidSharedLibrary.ts

```diff
@@ -0,0 +1,58 @@
+import { ConfigPlugin, withAppBuildGradle } from '@expo/config-plugins';
+
+/**
+ * Add pickFirst libraries into app/build.gradle to fix build errors.
+ * This is a dangerous operation. It will override packagingOptions {} or android { packagingOptions {} } - use with caution as it might break your app
+ * [Learn more](https://github.com/mrousavy/react-native-vision-camera/pull/545)
+ * Credits: @mercyaj [Credits](https://github.com/mrousavy/react-native-vision-camera/pull/545#issuecomment-949477264)
+ */
+
+export const withDangerouslyHandleAndroidSharedLibrary: ConfigPlugin = (c) => {
+  return withAppBuildGradle(c, (config) => {
+    if (config.modResults.language === 'groovy') {
+      const body = `
+        pickFirst 'lib/**/lib*.so'
+      `;
+      const regexpPackagingOptions = /\bpackagingOptions\s*{[^}]*}/;
```

Comment:
Consider reducing the complexity of this regex https://github.com/expo/config-plugins/issues/31#issuecomment-1009387444

Reply:
@EvanBacon I tried to reduce it, but other people came up with different .so's popping up (for me, it have been 4 in total, some had 2, some had 6). Marc suggested to use this glob pattern.

Reply:
Obsolete. Going to remove it